CONTRIBUTING.md

```diff
@@ -27,3 +27,47 @@ We are happy to see your enthusiasm. This table lists the criteria and how impor
 | Include core TF API aka no contrib   | Yes | Optional |
 | Include README     | Yes | Yes |
 | Maintain version dependency updates     | Yes | Optional |
+
+
+## Expectations
+
+### Contributors
+
+- Please read the Python Dev Guide and in particular the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+- Your Pull Request should be small and include a single logical change.
+
+- Smaller changes are easier to review and evaluate, and in general smaller self-contained PRs make it easier to debug or, if needed, rollback.
```

Comment:
I think the long term vision is also to have people contribute samples. Would this kinda expectation of small changes still apply?

Reply:
Agree, What If I reword it to be specific for existing samples?

"- Your Pull Request for existing samples should be small and include a single logical change."

Reply:
OOO yea, that'd be great.